### ISSUE TITLE
Category#getChannels doesn't return NewsChannels

### ISSUE THREAD

**Issue by V-Play-Games** [DV8FromTheWorld/JDA#2010]
[download]: https://bintray.com/dv8fromtheworld/maven/JDA/_latestVersion
[guild]: https://discord.gg/0hMr4ce0tIk3pSjp
[stack overflow]: https://stackoverflow.com/questions/tagged/java

## General Troubleshooting

  
- [x] I have checked for similar issues.
- [x] I have updated to the [latest JDA version][download].
- [x] I have checked the branches or the maintainers' PRs for upcoming bug fixes.

## Bug Report
Basically the title.
Category#getChannel doesn't return NewsChannels, and the docs don't include Stage Channels

### Expected Behavior
It should return News Channels

**Comment by Tais993** [comment]
What version are you using?

**Comment by V-Play-Games** [comment]
> What version are you using? (5 I'd assume)
5.0.0-alpha.5
